native-tokens/README.md

```diff
@@ -0,0 +1,12 @@
+# How to generate a trivial native-tokens policy
+
+1. Install IOG's `plutus`. After installing, run `nix-shell` at the
```

Comment:
What do you mean by installing IOG's plutus? Do you mean "clone IoG's Plutus repo"? If so, you may want to change the wording. Installing and cloning are different things

Reply:
Install.